cadc-util/src/main/java/ca/nrc/cadc/net/HttpDownload.java

```diff
@@ -458,27 +388,26 @@ private void runX()
             log.debug("caught: " + tex);
             throwTE = true;
             throw tex;
-        } catch (AccessControlException ex) {
+        } catch (AccessControlException | ExpectationFailedException | IllegalArgumentException | PreconditionFailedException | ResourceNotFoundException ex) {
```

Comment:
I'm sure there's a good reason for this set of 400 level errors to be caught here but I don't know what that is.

Reply:
removed these catch blocks